### ISSUE TITLE
ENOENT when invoking spawnSync on Node 0.10 / Ubuntu

### ISSUE THREAD

**Issue by mbleigh** [ForbesLindesay/spawn-sync#33]
ENOENT, no such file or directory '/tmp/spawn-sync_output_549843d802a98140c4b99d4af'

Receiving this with a pretty straightforward invocation of `spawn-sync`:

```js
var spawnSync = require('spawn-sync');
var result = spawnSync('my-command', {
  input: someString,
  timeout: 10000,
  encoding: 'utf-8'
});
```

Any ideas on what might be the cause?

**Comment by szydan** [comment]
I hit this error while executing my program on private node bin 
The node was not installed on the system and as the line 
```
https://github.com/ForbesLindesay/spawn-sync/blob/master/lib/spawn-sync.js#L85
```
require node to be installed I've got the same error as you.

**Comment by mbleigh** [comment]
Ah, yes, I'm using nvm which means node probably isn't in my path if a new shell is being spun up.

**Comment by ForbesLindesay** [comment]
Fixed by #35
